doc/private_endpoint_connection_command_guideline.md

```diff
@@ -0,0 +1,100 @@
+    --id                            : The ID of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). If specified --vault-name and --name/-n, this should be omitted.
+    --name -n                       : The name of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). Required if --id is not specified.
+    --resource-group -g             : Proceed only if Key Vault(Storage Account) belongs to the specified resource group.
+    --vault-name(--account-name)    : Name of the Key Vault(Storage Account). Required if --id is not specified.
+```
+- The `... private-endpoint-connection reject` command should look similar to the following, depending on which features are supported by the service.
+```
+Arguments
+    --description                   : Comments for the rejection.
+    --id                            : The ID of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). If specified --vault-name and --name/-n, this should be omitted.
+    --name -n                       : The name of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). Required if --id is not specified.
+    --resource-group -g             : Proceed only if Key Vault(Storage Account) belongs to the specified resource group.
+    --vault-name(--account-name)    : Name of the Key Vault(Storage Account). Required if --id is not specified.
+```
+- The `... private-endpoint-connection show/delete` command should look similar to the following, depending on which features are supported by the service.
+```
+Arguments
+    --id                            : The ID of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). If specified --vault-name and --name/-n, this should be omitted.
+    --name -n                       : The name of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). Required if --id is not specified.
+    --resource-group -g             : Proceed only if Key Vault(Storage Account) belongs to the specified resource group.
+    --vault-name(--account-name)    : Name of the Key Vault(Storage Account). Required if --id is not specified.
+```
+
+#### Private Link Resource
+
+- The parent resource should expose a single command group called `private-link-resource` with one commands: `list`.
+- The `... private-link-resource list` command should look similar to the following, depending on which features are supported by the service.
+
+```
+Arguments
+    --vault-name(--account-name) [Required] : Name of the Key Vault(Storage Account).
+    --resource-group -g     : Proceed only if Key Vault belongs to the specified resource group.
```

Comment:
Should we also support `--id` for PLR?

Reply:
My opinion is that we can just support `--vault-name`. We still follow our conversion. Another option is to support `--ids` by adding `id_part='name'`. It might cause a little confusion since it's a list command.

Reply:
> My opinion is that we can just support `--vault-name`. We still follow our conversion. Another option is to support `--ids` by adding `id_part='name'`. It might cause a little confusion since it's a list command.

Good point.